PROJECT_PROGRESSION.md

```diff
@@ -16,14 +16,71 @@ This governance policy sets forth the proposal process for projects to be accept
 
 New projects should express interest to join the OpenJS Foundation via an email to new-projects@lists.openjsf.org with a filled out [Project Application Template](./NEW_PROJECT_APPLICATION.md). The initial application will have a silent period for internal only discussion. If your application is denied during this phase you will be notified privately. If there are no objections to the application, projects will given an `incubation` status and begin the on-boarding process. **Note** the `incubation` status is temporary as we determine if the project is a good fit. At any time during the `incubation` process a project can withdraw either voluntarily or at the request of the CPC.
 
-#### Application Phases
+#### Roles
+
+##### Applicant
+
+Individual or group who submits the application on behalf of a project to join the OpenJS Foundation.
+
+##### Application Manager
+
+The application manager ensures progress through the process, helps the project understand the work to be completed and
+to move though the application process. Typically the application manager will be one of the Foundation staff members.
```

Comment:
I would lean into the language here a little more. "Typically" is ambiguous as to who _else_ is a valid Application Manager. In an attempt to be explicit I've suggested two other sets of folks to consider. Please feel free to accept that portion of the suggestion or not.

```md
The Application Manager ensures progress through the process, helps the project understand the work to be 
completed along with explaining where they are in the application process. Applications Managers can be a
member of the OpenJS Foundation staff, CPC Member, or other person approved by the CPC.
```